docs/stability.rst

```diff
@@ -251,6 +251,17 @@ The current planned and existing sub-packages are:
                 for the foreseeable future.
             </td>
         </tr>
+        <tr>
+            <td>
+                astropy.timeseries
+            </td>
+            <td align='center'>
+                <span class="mature"></span>
```

Comment:
Is "mature" an accurate categorization here? It conflicts with "in heavy development" just below.

Reply:
woops, copy pasta :spaghetti: